crates/subspace-networking/src/peer_info.rs

```diff
@@ -28,32 +27,11 @@ pub struct Notification;
 /// Defines a subscription to a peer-info notification.
 pub type NotificationHandler = Arc<dyn Fn(&Notification) + Send + Sync + 'static>;
 
-/// Cuckoo filter data transfer object.
-#[derive(Clone, Encode, Decode, Default)]
-pub struct CuckooFilterDTO {
-    /// Exported cuckoo filter values.
-    pub values: Vec<u8>,
-    /// Cuckoo filter items.
-    pub length: u64,
-}
-
-impl fmt::Debug for CuckooFilterDTO {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.debug_struct("CuckooFilterDTO")
-            .field("values", &self.length)
-            .field("length", &self.values.len())
-            .finish()
-    }
-}
-
 #[derive(Clone, Encode, Decode, Default, Debug)]
 /// Peer info data
 pub enum PeerInfo {
     /// DSN farmer.
-    Farmer {
-        /// Peer info data.
-        cuckoo_filter: Arc<CuckooFilterDTO>,
-    },
+    Farmer,
```

Comment:
Isn't this a breaking change on the network level?

Reply:
Yes. I added a related label.